rosdep/python.yaml

```
@@ -6678,6 +6678,13 @@ python3-jsonschema:
     '*': [python3-jsonschema]
     '7': [python36-jsonschema]
   ubuntu: [python3-jsonschema]
+python3-json-stream-pip:
+  debian:
+    pip:
+      packages: [json-stream]
+  ubuntu:
+    pip:
+      packages: [json-stream]
```

```suggestion
  '*':
    pip:
      packages: [json-stream]
  ```

For pip keys we are now taking advantage of an OS-level wildcard feature that was added to rosdep.